.github/workflows/mobilecoin-dev-cd.yaml

```diff
@@ -599,7 +599,8 @@ jobs:
             "namespace": "${{ needs.generate-metadata.outputs.namespace }}",
             "version": "${{ needs.generate-metadata.outputs.tag }}",
             "client_auth_enabled": "false",
-            "use_static_wallet_seeds": "true"
+            "use_static_wallet_seeds": "true",
+            "tokens_json_version": "2"
```

Comment:
So the tokens.json file format change regardless of whether we allow nested multisigs. The new format is compatible with the old, as long as you do not specify `multi_signers` and only stick to `individual_signers`.
I am not sure we'll bother testing nested multi-sigs in CI (our current governors are a simple 1-out-of-1 for CI). This can get covered in a Rust unit test or integration test.